tests/test_captions.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import json
+import os.path
+from requests.exceptions import HTTPError
+from src.rev_ai.models import Transcript, Monologue, Element
+from src.rev_ai.apiclient import RevAiAPIClient
+from tests.helpers.errors import get_error_test_cases
+
+try:
+    from urllib.parse import urljoin
+except ImportError:
+    from urlparse import urljoin
+JOB_ID = '1'
+URL = urljoin(RevAiAPIClient.base_url, 'jobs/{}/captions'.format(JOB_ID))
+
+
+@pytest.mark.usefixtures('mock_client', 'make_mock_response')
+class TestCaptionEndpoint():
+    def test_get_captions_with_success(self, mock_client, make_mock_response):
+        filename = 'TestFileToBeDeleted'
+        filepath = ''
+        data = 'Test'
+        response = make_mock_response(url=URL, text=data)
+        mock_client.session.get.return_value = response
+
+        res = mock_client.get_captions(JOB_ID, filename, filepath)
```

Comment:
Please make some negative tests for writing to file (ie. if no filename is specified then no file is created)